.editorconfig

```diff
@@ -3,6 +3,9 @@
 # SA1633: File should have header
 dotnet_diagnostic.SA1633.severity = none
 
+# SA1124: Do not use regions
```

Comment:
This is a debatable point, but I personally am not a fan of regions at all. I'd be concerned that by disabling the warning and putting them into the project it will set the precedent that regions are okay, and then we'll see actual implementations being obscured within regions.

Reply:
I know that regions are while contentious but I felt that with the size and number of responsibilities the `PackageTemplateBase` including them was just. 

Agreed, this does open up the entire project to using them although I think we can review these on a case by case basis to ensure that they aren't included unnecessarily.